README.md

```diff
@@ -559,6 +527,287 @@ You can build a pipeline to use [FHIR converter](https://github.com/microsoft/FH
 ### Can we use custom de-identification methods?
 Currently you can use the prebuilt de-identification methods and control their behavior by passing parameters. We are planning to support custom de-identification methods in future.
 
+# DICOM Data Anonymization
+
+The Digital Imaging and Communication in Medicine (DICOM) standard has been commonly used for storing, viewing, and transmitting information in medical imaging. A DICOM file not only contains a viewable image but also a header with a large variety of data elements. These meta-data elements include identifiable information about the patient, the study, and the institution. Sharing such sensitive data demands proper protection to ensure data safety and maintain patient privacy. DICOM Anomymization Tool helps anonymize metadata in DICOM files for this purpose.
+
+### Features
+- Support anonymization methods for DICOM metadata including redact, keep, encrypt, cryptoHash, dateShift, perturb, substitute, remove and refreshUID.
+- Configuration of the data elements that need to be anonymized.
+- Configuration of the anonymization methods for each data element.
+- Ability to run the tool on premise to anonymize a dataset locally.
+
+### Build the solution
+Use the .Net Core 3.1 SDK to build DICOM Anonymization Tool. If you don't have .Net Core 3.1 installed, instructions and download links are available [here](https://dotnet.microsoft.com/download/dotnet/3.1).
+
+### Prepare DICOM Data
+You can prepare your own DICOM files as input, or use sample DICOM files in folder $SOURCE\DICOM\samples of the project.
+
+## Anonymize DICOM data: using the command line tool
+
+Once you have built the command line tool, you will find executable file Microsoft.Health.Dicom.Anonymizer.CommandLineTool.exe in the $SOURCE\DICOM\src\Microsoft.Health.Dicom.Anonymizer.CommandLineTool\bin\Debug|Release\netcoreapp3.1 folder.
+
+You can use this executable file to anonymize DICOM file.
+
+```
+> .\Microsoft.Health.Dicom.Anonymizer.CommandLineTool.exe -i myInputFile -o myOutputFile
+```
+
+### Use Command Line Tool
+The command-line tool can be used to anonymize one DICOM file or a folder containing DICOM files. Here are the parameters that the tool accepts:
+
+
+| Option | Name | Optionality | Default | Description |
+| ----- | ----- | ----- |----- |----- |
+| -i | inputFile | Required (for file conversion) | | Input DICOM file. |
+| -o | outputFile | Required (for file conversion) | |  Output DICOM file. |
+| -c | configFile | Optional |configuration.json | Anonymizer configuration file path. It reads the default file from the current directory. |
+| -I | inputFolder | Required (for folder conversion) |  | Input folder. |
+| -O | outputFolder | Required (for folder conversion) |  | Output folder. |
+| --validateInput | validateInput | Optional | false | Validate input DICOM file against value multiplicity, value types and format in DICOM specification. |
```

Comment:
add link here for DICOM specification.
http://dicom.nema.org/medical/Dicom/2017e/output/chtml/part06/chapter_6.html